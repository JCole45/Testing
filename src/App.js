import React, {Component} from 'react';

    class App extends Component {
        showText(){
            return ("Hello World")
            
        }
     
      render () {
      
        return (
            <div>
           <p> {this.showText()}</p>
           </div>
        )
      }
    }

export default App 

