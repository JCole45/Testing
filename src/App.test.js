{/*import React from 'react';
import App from './App'
import renderer from 'react-test-renderer';

test('State must update', () => {
const component = renderer.create(
      <App page="Hello World"> Hello all</App>
    );

let tree = component.toJSON();
expect(tree).toMatchSnapshot();

tree.props.showText();
 tree = component.toJSON();
expect(tree).toMatchSnapshot();

}
)  */}
import React from 'react';
import { shallow } from 'enzyme';
import App from './App'


describe('First React component test with Enzyme', () => {
  it('renders without crashing', () => {
     shallow(<App />);
   });
});