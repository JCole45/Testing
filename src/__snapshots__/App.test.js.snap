// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`State must update 1`] = `
<div>
  <button
    onClick={[Function]}
  >
    click
  </button>
  <p>
     
    
  </p>
</div>
`;
